src/content/ui-api/kubectl-gs/template-nodepool.md

```diff
@@ -62,6 +62,8 @@ Here are the supported flags:
 ### Azure specific
 
 - `--azure-vm-size` - Azure VM size to use for workers (e.g. *Standard_D4s_v3*).
+- `--azure-spot-vms` - Wheter to use Spot VMs for this Node Pool or not (defaults to false that means not to use Spot VMs).
+- `azure-spot-max-price` - Max hourly price in USD to pay for one Spot VM. If the current price for the VM is exceeded, the VM is deleted.
```

Comment:
```suggestion
- `azure-spot-max-price` - Max hourly price in USD to pay for one spot VM. If the current price for the VM is exceeded, the VM is deleted.
```

Reply:
I'd like us to document what the default is. I. e. what price is used if this flag is not used.